From: Jon Bernard <jbernard@debian.org>
Date: Wed, 15 Dec 2010 10:07:37 -0500
Subject: [PATCH] Add force-reload action to initscripts
To: libcg-devel@lists.sourceforge.net

This change is required for Debian compatibility. Here is the relevant
lintian information:

    The /etc/init.d scripts have to support the following command line
    arguments: start, stop, restart, force-reload.

    Refer to Debian Policy Manual section 9.3.2 (Writing the scripts)
    for details.

I'm not sure if other distributions require this, but it probably
doesn't hurt.
---
 scripts/init.d/cgconfig.in |    4 ++--
 scripts/init.d/cgred.in    |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/scripts/init.d/cgconfig.in b/scripts/init.d/cgconfig.in
index 5d1d0d5..ccd779c 100644
--- a/scripts/init.d/cgconfig.in
+++ b/scripts/init.d/cgconfig.in
@@ -160,7 +160,7 @@ trapped() {
 }
 
 usage() {
-    echo "$0 <start|stop|restart|condrestart|status>"
+    echo "$0 <start|stop|restart|force-reload|condrestart|status>"
     exit 2
 }
 
@@ -185,7 +185,7 @@ case $1 in
         start;
         RETVAL=$?
         ;;
-    'restart')
+    'restart'|'force-reload')
         common
 	stop
         start
diff --git a/scripts/init.d/cgred.in b/scripts/init.d/cgred.in
index a12c5fe..0401395 100644
--- a/scripts/init.d/cgred.in
+++ b/scripts/init.d/cgred.in
@@ -123,7 +123,7 @@ case "$1" in
 		status -p $pidfile $servicename
 		RETVAL=$?
 		;;
-	restart)
+	restart|force-reload)
 		stop
 		start
 		;;
@@ -148,7 +148,7 @@ case "$1" in
 		fi
 		;;
 	*)
-		echo $"Usage: $0 {start|stop|status|restart|condrestart|reload}"
+		echo $"Usage: $0 {start|stop|status|restart|force-reload|condrestart|reload}"
 		RETVAL=2
 		;;
 esac
-- 
